gson/src/main/java/com/google/gson/stream/JsonReader.java

```diff
@@ -294,17 +333,15 @@ public JsonReader(Reader in) {
 
   /**
    * Configure this parser to be liberal in what it accepts. By default,
-   * this parser is strict and only accepts JSON as specified by <a
-   * href="http://www.ietf.org/rfc/rfc4627.txt">RFC 4627</a>. Setting the
+   * this parser is semi-strict and only accepts JSON as specified by <a
+   * href="https://www.ietf.org/rfc/rfc8259.txt">RFC 8259</a>. Setting the
```

Comment:
This is slightly misleading because it is more lenient (="semi-strict") than RFC 8259, though this might not matter here much

Reply:
Ugh. Good catch! I removed the "only" part. Now, it says that it "accepts RFC 8259
(including some slight variations)".
I just did a forced push.

Thanks a lot for your (initial) review! Much appreciated!